genesis/genesis_camino.json

```
@@ -1,62 +1,96 @@
 {
   "networkID": 1000,
+  "allocations": null,
+  "startTime": 1166713120,
```

```suggestion
  "startTime": 1667131200,
```
Looks like startTime is wrong.